internal/support/netlog.py

```diff
@@ -916,6 +916,8 @@ def process_url_request_event(self, event):
             if 'has_raw_bytes' not in entry or not entry['has_raw_bytes']:
                 entry['bytes_in'] += params['byte_count']
                 entry['chunks'].append({'ts': event['time'], 'bytes': params['byte_count']})
+            elif entry['chunks']:
+                entry['chunks'][-1]['filtered'] = params['byte_count']
```

Comment:
_Super_ minor nit, but why 'filtered'? Would 'uncompressed' or something like that make more sense?

Reply:
It is repeated in every chunk entry for every request so I was trying to keep it as short as possible (though I'm sure it will compress well since it is repeated).  I just picked "filtered" since it is part of "URL_REQUEST_JOB_FILTERED_BYTES_READ" but I'm happy to pick something more descriptive.

Happy to switch it to "uncompressed" if I was just being too cautious with the concern of the JSON size.

Reply:
Maybe "inflated" or "decoded"?

Reply:
Switched it to inflated - hopefully people won't infer that the request was using "deflate" compression but it's the same length as filtered and hopefully clearer.

Reply:
Works for me!